change/@fluentui-react-components-2c418aac-078d-45dc-ad47-4bf796a2e600.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32247&commitId=9764b7a742a99ad06922e3d440aebe8b4242a94e&env=prod&clientType=FLUENTUI)


  <details><summary><b>TagPicker</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.secondary action - Dark Mode.chromium.png|116| Changed|
|TagPicker.size - High Contrast.chromium.png|75| Changed|
|TagPicker.secondary action - RTL.chromium.png|128| Changed|
|TagPicker.secondary action.chromium.png|128| Changed|
|TagPicker.size - Dark Mode.chromium.png|38| Changed|
|TagPicker.size.chromium.png|47| Changed|
|TagPicker.secondary action - High Contrast.chromium.png|172| Changed|
|TagPicker.size - RTL.chromium.png|47| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>